storage/postgres/migrations/20190913161000_paging.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+ALTER TABLE brokers ADD COLUMN paging_sequence SERIAL;
+ALTER TABLE platforms ADD COLUMN paging_sequence SERIAL;
+ALTER TABLE visibilities ADD COLUMN paging_sequence SERIAL;
```

Comment:
While serial (4 bytes) is a big number over time it might not be sufficient, especially with visibilities. Wouldn't it be better to use bigserial (8 bytes)?
Also all variables "PagingSequence" should be int64.